fs/sync/sync.go

```diff
@@ -756,6 +756,10 @@ func (s *syncCopyMove) run() error {
 	// Read the error out of the context if there is one
 	s.processError(s.ctx.Err())
 
+	if accounting.Stats(s.ctx).GetBytes() == 0 {
```

Comment:
I think this should probably say `GetTransfers() == 0` just in case we are transferring 0 length files